examples/deployment/assets/3-gold-msig.multilsig

```diff
@@ -0,0 +1,33 @@
+// This file was created using goal by doing
+//   - create a new multisignature account (goal account multisig new ..)
+//   - create logic signature and sign the logic signature using `goal clerk multisig signprogram..`
+//   - output of Decompiling the logic signature is pasted in this file
+// reference: https://developer.algorand.org/docs/features/asc1/stateless/walkthrough/#creating-a-multi-signature-delegated-logic-sig
+// version 1
+intcblock 1 100
+txn TypeEnum
+intc_0
+==
+txn Amount
+intc_1
+<=
+&&
+LogicSig: {
```

Comment:
this is the file generated from `goal` .
and it has whole program

Reply:
@ratik21

Reply:
In this file, the LogicSig: Object contains only the msig field. That's the reason i am creating the lsig (`loadLsig(..)`) and then attaching the msig fetched from here to the lsig generated (to get the `Logic`, `sig`, 'Args`). Will read more.

Reply:
i think i am not explaining it clear here, let's discuss it on tommorow's call

Reply:
Noted.